nautobot/core/views/renderers.py

```diff
@@ -184,6 +185,9 @@ def get_context(self, data, accepted_media_type, renderer_context):
                 form = form_class(initial=request.GET)
             elif view.action in ["create", "update"]:
                 initial_data = normalize_querydict(request.GET, form_class=form_class)
+                # Manually populate the reverse side of the manytomany relationship in ContactForm
+                if isinstance(instance, Contact):
+                    initial_data["teams"] = instance.teams.all()
```

Comment:
Do we similarly need to load `"contacts"` if `instance` is a `Team`? Or is that handled automatically as the forward side of the M2M?

Reply:
Handled automatically.